projects/storefrontapp-cds-integration/src/environments/environment.ci.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  occBaseUrl: 'https://dev-com-17.accdemo.b2c.ydev.hybris.com:9002',
+  occBaseUrl: 'https://dev-com-14.accdemo.b2c.ydev.hybris.com:9002',
```

Comment:
leave it to 17, but I am guessing you are using the start-open script, which rebuilds the libs, shellapp, and runs the e2e against it.

But what you can do is
- `yarn start:cds`
- `yarn e2e:cy:open` --> headless just to verify it is doing what it is supposed to do in the UI
- `yarn e2e:cy:run --spec cypress/integration/vendor/cds/** for all 
OR
- `yarn e2e:cy:run --spec cypress/integration/vendor/cds/profile-tag.e2e-spec.ts to run in headless. 
^ it's best to run it in headless because it the UI version sometimes fail due to it being too slow or too fast.

Reply:
I thought only dev-14 had the cds useage?

Reply:
as discussed, leave it to 17, because the pipeline will run it on 17 soon